contrib/unetstack3-location/org/subnero/unet/nodeinfo/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * Node information agents from Subnero.
+ */
+package org.subnero.unet.nodeinfo;
```

Comment:
wondering whether it should be `com.subnero.unet.nodeinfo` instead of `org`.

Reply:
That's what the original was. We can change it if required.